### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by chrisdemars** [travisneilson/DevTipsShowWebsite#10]
Hey Trav, I noticed some elements like img tags are missing alt attr's. I would like to work on some a11y stuff for the Dev Tips site.

**Comment by travisneilson** [comment]
Is this what you tweeted me about? Have at it!!

**Comment by chrisdemars** [comment]
Ha yeah. Didn't want to do anything unless u said it was totally cool.
